src-docs/src/views/combo_box/combo_box_example.js

```diff
@@ -567,5 +573,28 @@ export const ComboBoxExample = {
       snippet: startingWithSnippet,
       demo: <StartingWith />,
     },
+    {
+      title: 'Duplicate labels',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: duplicateOptionsSource,
+        },
+        {
+          type: GuideSectionTypes.HTML,
+          code: duplicateOptionsHtml,
+        },
+      ],
+      text: (
+        <p>
+          If you want to use options with a duplicate label (which in general is
+          not recommended since the user has no way to distinguish those
+          options), you need to set a unique <EuiCode language="js">id</EuiCode>{' '}
+          for each option.
+        </p>
+      ),
+      props: { EuiComboBox },
+      demo: <DuplicateOptions />,
```

Comment:
Then add the snippet here:

```suggestion
      demo: <DuplicateOptions />,
      snippet: duplicateOptionsSnippet,
```